lib/Slider/Slider.js

```diff
@@ -269,19 +269,14 @@ module.exports = kind(
 	* @private
 	*/
 	jumpWrapperComponents: [
-		{name: 'buttonLeft', kind: IconButton, backgroundOpacity: 'transparent', classes: 'moon-slider-button left', icon: 'arrowlargeleft', onSpotlightKeyDown: 'configureSpotlightHoldPulse', onSpotlightSelect: 'previous', ondown: 'downLeft', onholdpulse: 'holdLeft', ondragstart: 'preventDrag', defaultSpotlightDisappear: 'buttonRight'},
+		{name: 'buttonLeft', kind: IconButton, backgroundOpacity: 'transparent', classes: 'moon-slider-button left', icon: 'arrowlargeleft', onSpotlightKeyDown: 'showKnobStatus', onSpotlightKeyUp: 'hideKnobStatus', onSpotlightSelect: 'previous', ondown: 'downLeft', onup: 'hideKnobStatus', onholdpulse: 'holdLeft', ondragstart: 'preventDrag', defaultSpotlightDisappear: 'buttonRight'},
```

Comment:
Oh, I didn't expect that you reply at this moment :+1: 
I'm so happy if you check it. Thank you!